src/js/widgets/SelectLocation/SelectLocationWidget.jsx

```diff
@@ -50,7 +51,7 @@ export default class SelectLocationWidget extends Component {
       };
 
     this.state = {
-      center: [location.position.lng || -97.7460479736328, location.position.lat || 30.266184073558826],
+      center: [location.position.lng, location.position.lat],
```

Comment:
do we have defaults? if not how are we handling this?

Reply:
I moved that inside of the onStyleLoad function: https://github.com/cityofaustin/us-forms-system/blob/1819edab1f496af299d86a68ae8948de6919fbd5/src/js/widgets/SelectLocation/SelectLocationWidget.jsx#L126

Thinking that it is really just supposed to be the default starting point, so having that in initial state seems conceptually weird to me ( and was initially part of me debugging the formData for location, though in retrospect with the final fix I don't think it was necessary )